src/backend/routers/chat.py

```diff
@@ -172,6 +178,12 @@ def process_chat(
     """
     user_id = request.headers.get("User-Id", "")
     deployment_name = request.headers.get("Deployment-Name", "")
+    model_config = {}
+    if not request.headers.get("Deployment-Config", "") == "":
+        model_config = dict(
```

Comment:
It's unclear to me what `Deployment-Config` input should be, and what it becomes when assigned to `model_config`, would recommend adding some comments here to clarify

Reply:
If I understand the code, it looks like we want to add a config dictionary we can pass to request headers to define a model deployment, which we can then call during the request (without having to preregister variables in the .env), is my understanding correct?

Reply:
Yes that is correct, so you can set model config per request